rpc/src/common.rs

```diff
@@ -0,0 +1,15 @@
+use {
+    solana_runtime::bank::Bank,
+    solana_sdk::{account::AccountSharedData, pubkey::Pubkey},
+    std::collections::HashMap,
+};
+
+pub(crate) fn get_account(
```

Comment:
Let's make this fn name more descriptive:
```suggestion
pub(crate) fn get_account_from_overwrites_or_bank(
```

Reply:
fixed [e30e65a](https://github.com/solana-labs/solana/pull/34619/commits/e30e65a9ef565d97bd8462a034839800f855cdd3)